CHANGELOG.md

```diff
@@ -1,3 +1,47 @@
+# 1.7.0-alpha03 (August 2024)
+
+_Changes since 1.7.0-alpha02_
+
+## Highlights
+### Multiple Platforms
+- [Skiko version was updated to 0.8.11 (It is based on skia m126 release)](https://github.com/JetBrains/compose-multiplatform-core/pull/1486)
+
+## Features
+### Multiple Platforms
+- [Commonized `material3-window-size-class` module](https://github.com/JetBrains/compose-multiplatform-core/pull/1466)
+- [Commonized `material3-common` module](https://github.com/JetBrains/compose-multiplatform-core/pull/1465)
+- [Commonized `:compose:material3:adaptive:adaptive`, `:compose:material3:adaptive:adaptive-layout`, `:compose:material3:adaptive:adaptive-navigation` and `:window:window-core` modules](https://github.com/JetBrains/compose-multiplatform-core/pull/1468)
+
+### iOS
+- [Opt-out of new touches strategy on iOS](https://github.com/JetBrains/compose-multiplatform-core/pull/1460)
```

Comment:
It's not valid

@elijah-semyonov please remove it from description (not just apply strikethrough formatting)